### PR TITLE
Improving the dashboard

### PR DESCRIPTION
**Pull request by fgregoribattista**
fix tweened numbers and add link to rsk swap

### CODE REVIEW

File: src/styles/_supplyBorrow.scss [hunk 1]
```diff
@@ -22,4 +22,20 @@
       line-height: 24px;
     }
   }
+}
+
+.rsw-swap-launch{
```
Comment (pmprete) [inline src/styles/_supplyBorrow.scss:27]: rename to token-bridge

File: src/styles/_supplyBorrow.scss [hunk 2]
```diff
@@ -22,4 +22,20 @@
...
+  a{
+
+    text-decoration:none !important;
+    color:black!important;
+    font-family: "Rubik", sans-serif;
+    font-style: normal;
+  }
+}
+.rsw-swap-launch-logo{
```
Comment (pmprete) [inline src/styles/_supplyBorrow.scss:36]: same here

File: src/views/MyActivity.vue [hunk 3]
```diff
@@ -1,6 +1,6 @@
 <template>
   <div class="my-activity">
-    <div class="upper-banner">
+    <div v-if="dataLaoded" class="upper-banner">
```
Comment (pmprete) [inline src/views/MyActivity.vue:3]: small typo, insteda of dataLaoded should be dataLoaded

File: src/views/MyActivity.vue [hunk 4]
```diff
@@ -217,6 +243,8 @@ import { mapState } from 'vuex'
 import { cTokensDetails } from '../middleware/constants'
 import Vue from 'vue'
 import VueNumber from 'vue-number-animation'
+Vue.config.productionTip = false
```
Comment (pmprete) [inline src/views/MyActivity.vue:246]: is this value used?

File: src/views/MyActivity.vue [hunk 5]
```diff
@@ -228,14 +256,15 @@ export default {
...
-      oldTotalSupplied: 0,
-      totalBorrowed: 0,
+      totalBalance: [],
+      totalSupplied: [],
+      totalBorrowed: [],
       showHealthWarning: null,
       polling: null,
       markets: [],
       accountStorage: '',
+      auxiliar: 1,
```
Comment (pmprete) [inline src/views/MyActivity.vue:266]: rename to something more clear please

File: src/views/SupplyBorrow.vue [hunk 6]
```diff
@@ -59,6 +59,17 @@
...
     </v-row>
+    <v-row class="d-flex justify-center rsw-swap-launch">
+      <v-banner elevation="0"
+        ><v-avatar> </v-avatar>
+        <a target="_blank" :href="getHttpTokenBridge">{{
+          currentComponent == 'SupplyList'
+            ? 'Bring your ethereum tokens '
+            : 'Cross your tokens to ethereum '
+        }}</a>
+        <v-img class="rsw-swap-launch-logo" :src="require(`@/assets/rsk_logo.svg`)" />
```
Comment (pmprete) [inline src/views/SupplyBorrow.vue:70]: use token bridge logo, it can be a jpg instead of a svg

**Review by pmprete — CHANGES_REQUESTED** [review]
Except my comments looks good

File: src/styles/_supplyBorrow.scss (hunk 1 above)
Comment (fgregoribattista) [reply src/styles/_supplyBorrow.scss:27]: Done!

File: src/views/MyActivity.vue (hunk 3 above)
Comment (fgregoribattista) [reply src/views/MyActivity.vue:3]: Done!

File: src/views/MyActivity.vue (hunk 4 above)
Comment (fgregoribattista) [reply src/views/MyActivity.vue:246]: not use, I remove it.

File: src/views/MyActivity.vue (hunk 5 above)
Comment (fgregoribattista) [reply src/views/MyActivity.vue:266]: not use, I remove it.

File: src/views/SupplyBorrow.vue (hunk 6 above)
Comment (fgregoribattista) [reply src/views/SupplyBorrow.vue:70]: Done!

**Review by pmprete — APPROVED** [review]
LGTM

**Pull request merged**